### PR TITLE
Update the contributor

### PR DESCRIPTION
**Pull request by imback82**
### What is the context for this pull request?

Update the contributor list to add a new contributor.

### Does this PR introduce _any_ user-facing change?

No

### How was this patch tested?

N/A

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -179,7 +179,12 @@ ThisBuild / developers := List(
     id = "thugsatbay",
     name = "Gurleen Singh",
     email = "",
-    url = url("https://github.com/thugsatbay")))
+    url = url("https://github.com/thugsatbay")),
+  Developer(
+    id = "clee704",
+    name = "Chungmin Lee",
+    email = "",
+    url = url("https://github.com/clee704")))
```
Comment (imback82) [inline build.sbt:187]: @clee704 Can you verify this info is correct?

**Review by clee704 — APPROVED** [review]
LGTM, thanks!

**Comment by andrei-ionescu** [comment]
@rapoth, @imback82, @clee704 Is this only a Microsoft dev list? 😁

**Comment by imback82** [comment]
Sorry, the title is misleading. Yes, this is an internal dev list for now, but we may expand it in the future.

**Pull request merged**